src/commands/clone.ts

```diff
@@ -37,10 +21,13 @@ export default async (scriptId: string, versionNumber: number, cmd: { rootDir: s
   scriptId = scriptId ? extractScriptId(scriptId) : await getScriptId();
   spinner.setSpinnerTitle(LOG.CLONING);
   const rootDir = cmd.rootDir;
-  saveProject({
-    scriptId,
-    rootDir,
-  }, false);
+  saveProject(
```

Comment:
Can we undo this? Is it required?

Reply:
I can manually reformat as

```ts
  saveProject({
    scriptId,
    rootDir,
  }, false);
```

but next `npm run prettier` will change it again (unless we tweak prettier config maybe)